.github/workflows/publish.yaml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
     branches: [ main ]
   push:
-    tags: [ '[A-z]+-v[0-9]+.[0-9]+.[0-9]+' ]
+    tags: [ '[A-z0-9]+-v[0-9]+.[0-9]+.[0-9]+' ]
```

Comment:
Do document the *intent* of the regexp. Which strings should it accept.
Never have an undocumented RegExp. It may be wrong, but no reader can tell if they don't know what it is trying to do.